Formula/b/bootterm.rb

```diff
@@ -0,0 +1,19 @@
+class Bootterm < Formula
+  desc "Simple, reliable and powerful terminal to ease connection to serial ports"
+  homepage "https://github.com/wtarreau/bootterm"
+  url "https://github.com/wtarreau/bootterm/archive/refs/tags/v0.5.tar.gz"
+  sha256 "95cc154236655082fb60e8cdae15823e4624e108b8aead59498ac8f2263295ad"
+  license "MIT"
+  head "https://github.com/wtarreau/bootterm.git", branch: "master"
+
+  depends_on xcode: :build
+
+  def install
+    bin.mkdir
+    system "make", "PREFIX=#{prefix}", "install"
+  end
+
+  test do
+    system bin/"bt", "-h"
```

Comment:
We prefer test that exercise a bit more features than the help command. How about this?
```suggestion
    assert_match "port", shell_output("#{bin}/bt -l")
```